cura/UI/MachineSettingsManager.py

```diff
@@ -0,0 +1,72 @@
+
+from typing import Any, Dict, Optional, TYPE_CHECKING
+
+from PyQt5.QtCore import Qt, QObject, pyqtSlot
+
+from UM.i18n import i18nCatalog
+
+
+class MachineSettingsManager(QObject):
+
+    def __init__(self, parent: Optional["QObject"] = None) -> None:
+        super().__init__(parent)
+        self._i18n_catalog = i18nCatalog("cura")
+
+        from cura.CuraApplication import CuraApplication
```

Comment:
Is it still possible to patch the application if we handle it like this? Otherwise, we might get into trouble with writing tests.

Reply:
Fixed in f4d0e3978